ast/astutil/format_test.go

```diff
@@ -90,8 +90,7 @@ func TestFormatWithCommentsParens(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if want := `
-// comment\n(1 * 1)`; want != got {
+	if want := `// comment\n(1 * 1)`; want != got {
```

Comment:
Is this a change that was added to the formatter or just a go format change?

Reply:
We made this go format change, we deemed it acceptable behavior as it stands.

Reply:
Cool! if its the go formatter, then it seems fine.